lmcache/v1/lookup_client/lmcache_lookup_client.py

```diff
@@ -60,6 +74,11 @@ def __init__(self, vllm_config: "VllmConfig"):
     def lookup(self, token_ids: torch.Tensor, request_id: Optional[str] = None) -> int:
         token_bufs = self.encoder.encode(token_ids)
         request_id_buf = request_id.encode("utf-8")
+        if self.create_lookup_server_only_on_worker_0:
+            self.socket.send_multipart(token_bufs + [request_id_buf], copy=False)
+            resp = self.socket.recv()
+            result = int.from_bytes(resp, "big")
+            return result
         results = []
         for i in range(self.tensor_parallel_size):
```

Comment:
We can do similar thing as above:

```suggestion
        ranks = self.tensor_parallel_size
        if self.create_lookup_server_only_on_worker_0:
            ranks = 1
        results = []
        for i in range(ranks):
```

Reply:
good suggestion!